azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -243,14 +217,11 @@ public Project getProject(String name) {
      */
     public Project getProject(int id) {
         Project fetchedProject = null;
-        if (isActiveProject(id)) {
-            fetchedProject = projectsById.get(id);
-        } else {
-            try {
-                fetchedProject = projectLoader.fetchProjectById(id);
-            } catch (ProjectManagerException e) {
-                logger.error("Could not load project from store.", e);
-            }
+        try {
+            fetchedProject = projectLoader.fetchProjectById(id);
+            loadAllProjectFlows(fetchedProject);
```

Comment:
What does load all project flows do? Are there cases where I need the project and not load the flows?

If they need to always be done together. they should be encapsulated in a method probably in `ProjectLoader` if appropriate.